docs/translating/polish.md

```diff
@@ -190,21 +190,21 @@ You can pin streams in the left-side panel.
 
 > Możesz przypiąć wybrane kanały w menu kontekstowym po lewej.
 
-**private message**: prywatna wiadomość
+**Direct Message**: Direct Message
```

Comment:
Looks like we don't have a translation for Direct Message to Polish in Transifex either.

Reply:
Ya I know that and that's why I did the same thing as I did in HIndi translation.

I'll remove these changes !! Today